### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by zdc217**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ This is the source code for an API allowing management of users that have been p
     ```
     to build the images again. 
 
+    If you want to upgrade the packages in the requirements files run:
+    ```
+    > make upgrade
+    > make
```
Comment (edthedev) [inline README.md:64]: This is a great command.

**Review by edthedev — APPROVED** [review]
I love this update.

**Review by baristaTam — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MKletz) [reply README.md:64]: FYI github supports syntax highlighting for Make. "```Makefile"

Comment (akibare999) [reply README.md:64]: This indeed does look amazing. Wow.

**Review by akibare999 — APPROVED** [review]
Like I said this is amazing.

**Pull request merged**